python/test/unit/language/test_compile_errors.py

```diff
@@ -341,7 +341,7 @@ def kernel(a=1, B: tl.constexpr = ""):
     triton.compile(triton.compiler.ASTSource(fn=kernel, signature={'a': 'i32'}, constants={'B': ""}))
 
 
-def test_where_warning():
+def test_where_warning(monkeypatch):
```

Comment:
can we use the existing `fresh_triton_cache`

Reply:
Thank you so much! I didn't know about this option, but it's a great fit.
I have updated the PR.